### ISSUE TITLE
System mails sent without a 'From' header

### ISSUE THREAD

**Issue by j-a-pope** [litespeedtech/cyberpanel#513]
Some system mails, are being sent without a 'From' header, which means they are either rejected by or sent to spam by receiving mail servers. 

This definitely includes "Disk Usage Warning" mails and may also be others. 

Issue started from forum post: https://forums.cyberpanel.net/discussion/5032/cp-sending-some-emails-without-a-from-header

**Comment by meramsey** [comment]
this is kinda whack

looks like notification generated here?
https://github.com/usmannasir/cyberpanel/blob/stable/plogical/IncScheduler.py#L178-L209#L178-L209

then uses the below mismatched stuff?
https://github.com/usmannasir/cyberpanel/blob/stable/ApachController/BackupUtil.py
https://github.com/usmannasir/cyberpanel/blob/stable/IncBackups/IncBackupsControl.py#L825-L840

@usmannasir @aonsyed 

Would it be possible to maybe use Django builtin libs and classes and nice wrapper function so theres a standard way to send and email alert from
https://docs.djangoproject.com/en/3.1/topics/email/
https://docs.python.org/3/library/email.examples.html

cyberpanel@hostname via localhost SMTP.

basically upon installation a nonvisible cyberpanel@hostname or similiar  account is made and credentials used so it can send out valid formatted notices which show smtp authd?

this would alleviate tons of missed notices when stuff is sent out like phpmail like style with no auth and missing or malformatted headers.

**Comment by meramsey** [comment]
So it looks like there is a class for emailing but its not being used cause the reference here links to `logging.SendEmail(admin.email, admin.email, finalText, message)`

![image](https://user-images.githubusercontent.com/1596188/107106923-2ed40b00-67fc-11eb-9cd1-224b7e7ba209.png)


vs to mailUtilities.SendEmail
https://github.com/usmannasir/cyberpanel/blob/599ddf46c105e27f4f93b3d4ff8f0b96d27d3455/plogical/mailUtilities.py#L36

Maybe its as simple as changing it from 
logging.SendEmail

to
mailUtilities.SendEmail

For all the code references like that and fixing up the hardcoded addresses in
https://github.com/usmannasir/cyberpanel/blob/599ddf46c105e27f4f93b3d4ff8f0b96d27d3455/ApachController/BackupUtil.py#L21-L23

See also this:
https://github.com/usmannasir/cyberpanel/pull/503/files

**Comment by jordanlambrecht** [comment]
@John-S4 did #503 fix this? If so, can you please close this issue out?

**Comment by j-a-pope** [comment]
> @John-S4 did #503 fix this? If so, can you please close this issue out?

I'm assuming yes, but personally I didn't have a chance to check it yet.

**Comment by TimSauer94** [comment]
Has this Issue been fixed with 2.1.1.? Can anyone confirm?
I am still using 2.0.3 for the moment...
